subt/script/rear-artf.bat

```diff
@@ -0,0 +1,4 @@
+cd artf
+python -m osgar.replay --module detector %1 --config m:\git\osgar\subt\script\rear-k2-sensors.json -F
```

Comment:
Assuming from the `cd artf` part, this script is run from a very specific directory. Is that correct? If so, could the `--config` part be made relative to that directory? Otherwise the script is limited to the specific layout and disk labels (which might be ok?).

Reply:
yes, it is all hard-coded, including absolute path, which here can be removed